internal/proxy/oauthproxy.go

```diff
@@ -533,6 +566,13 @@ func (p *OAuthProxy) RobotsTxt(rw http.ResponseWriter, _ *http.Request) {
 	fmt.Fprintf(rw, "User-agent: *\nDisallow: /")
 }
 
+// Certs publishes the public key necessary for upstream services to validate the digital signature
+// used to sign each request.
+func (p *OAuthProxy) Certs(rw http.ResponseWriter, _ *http.Request) {
+	rw.WriteHeader(http.StatusOK)
+	fmt.Fprint(rw, p.publicCertsJSON)
```

Comment:
We don't need to convert this to a string to use it here, nor do we _have_ to call WriteHeader, we could just:


```golang
rw.Write(p.RawPublicCerts)
```

Which will default to `200` and write our string as bytes.

Reply:
Nice, good suggestion. Thanks!